app/src/main/res/values/strings.xml

```diff
@@ -113,4 +113,10 @@
     <string name="status_disconnecting">disconnecting</string>
     <string name="status_waiting">connecting in %d seconds</string>
 
+    <string name="menu_chatroom_sort">Sort</string>
```

Comment:
I wonder if you should also provide at least "Google Translate" versions of these strings in the other supported languages. I'd think it'd be better than having missing strings, right?

Reply:
Sure @bizzbyster I will add translation after implementing complete feature :smile: Thanks!!!